manifests/logind.pp

```diff
@@ -27,4 +27,8 @@
       }
     }
   }
+
+  if !(empty($systemd::loginctl_users) {
+    create_resources(loginctl_user, $systemd::loginctl_users)
```

Comment:
It's a bit redundant to check for an empty hash. These days it's better to avoid `create_resources`. There may be typos in the following code, but roughly like this:
```suggestion
  $systemd::loginctl_users.each |$loginctl_name, $loginctl_user| {
    loginctl_user { $loginctl_name:
      * => $loginctl_users,
```

Reply:
I'll merge this in with some minor tweaks.